docs/howto/garbage-collection/internals.md

```diff
@@ -1,15 +1,23 @@
 ---
-title: "Internals: Committed GC"
+title: Internals
```

Comment:
Please note that @johnnyaug has a PR (might be already merged) with changes to the GC documentation